keps/sig-api-machinery/2876-crd-validation-expression-language/README.md

```diff
@@ -1168,30 +1246,26 @@ combination of large, compact, complex vs. similar combinations using existing v
 <!--
 This section must be completed when targeting beta to a release.
 
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+
 The Troubleshooting section currently serves the `Playbook` role. We may consider
 splitting it into a dedicated `Playbook` document (potentially with some monitoring
 details). For now, we leave it here.
 -->
 
 ###### How does this feature react if the API server and/or etcd is unavailable?
 
+Same as without the feature.
+
 ###### What are other known failure modes?
 
-<!--
-For each of them, fill in the following information by copying the below template:
-  - [Failure mode brief description]
-    - Detection: How can it be detected via metrics? Stated another way:
-      how can an operator troubleshoot without logging into a master or worker node?
-    - Mitigations: What can be done to stop the bleeding, especially for already
-      running user workloads?
-    - Diagnostics: What are the useful log messages and their required logging
-      levels that could help debug the issue?
-      Not required until feature graduated to beta.
-    - Testing: Are there any tests for failure mode? If not, describe why.
--->
+N/A
 
 ###### What steps should be taken if SLOs are not being met to determine the problem?
 
+We have added resource constraints to the feature. That being said, the request will fail if any particular expression is costly or failing or the entire request exceed certain limit.
```

Comment:
is there a metric for this and is it per-resource or just in aggregate?

Reply:
We have added a cost limit per-resource and a cost limit per expression as well. Ref: https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/2876-crd-validation-expression-language/README.md#resource-constraints